dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -131,6 +133,12 @@ public class ConnectorArguments extends DefaultArguments {
     private final OptionSpec<String> optionDatabase = parser.accepts(OPT_DATABASE, "Database(s) to export").withRequiredArg().ofType(String.class).withValuesSeparatedBy(',').describedAs("db0,db1,...");
     private final OptionSpec<String> optionSchema = parser.accepts(OPT_SCHEMA, "Schemata to export").withRequiredArg().ofType(String.class).withValuesSeparatedBy(',').describedAs("sch0,sch1,...");
     private final OptionSpec<Void> optionAssessment = parser.accepts(OPT_ASSESSMENT, "Whether to create a dump for assessment (i.e., dump additional information).");
+
+    private final OptionSpec<Long> optionMaxTableSizeVRows = parser.accepts(OPT_MAX_TABLESIZEV_ROWS,
```

Comment:
You maybe also want to add an annotation to the TeradataMetadataConnector to indicate that it respects this parameter. Then this option will be shown in the connector-related documentation in `--help` output.

See `@RespectsArgumentDatabasePredicate` as an example

Reply:
Done.